tests/template_backends/test_django.py

```diff
@@ -145,3 +146,32 @@ def test_non_debug_default_template_loaders(self):
     def test_debug_default_template_loaders(self):
         engine = DjangoTemplates({'DIRS': [], 'APP_DIRS': True, 'NAME': 'django', 'OPTIONS': {}})
         self.assertEqual(engine.engine.loaders, self.default_loaders)
+
+    @override_settings(DEBUG=True)
+    def test_debug_template_loaders_with_dirs(self):
+        engine = DjangoTemplates({
+            'DIRS': ['dirs'],
+            'APP_DIRS': True,
+            'NAME': 'django',
+            'OPTIONS': {},
+            'POST_APP_DIRS': ['post_app_dirs'],
+        })
+        self.assertEqual(engine.dirs, ['dirs'])
+        self.assertEqual(
+            engine.engine.loaders, [
+                'django.template.loaders.filesystem.Loader',
```

Comment:
It could be changed later but requires some test refactoring, see https://github.com/django/django/blob/412997f8baff8df934709b9520c9863d629c687f/tests/template_tests/test_loaders.py#L285-L292.